src/main/scala/org/ergoplatform/nodeView/history/extra/ExtraIndexSerializer.scala

```diff
@@ -1,9 +1,9 @@
 package org.ergoplatform.nodeView.history.extra
 
-import scorex.core.serialization.ScorexSerializer
+import scorex.core.serialization.ErgoSerializer
 import scorex.util.serialization.{Reader, Writer}
 
-object ExtraIndexSerializer  extends ScorexSerializer[ExtraIndex]{
+object ExtraIndexSerializer  extends ErgoSerializer[ExtraIndex]{
```

Comment:
I wonder if we need scaladocs for serializers? I don't think they are required, but maybe you do.

Reply:
The reason I think it is not required because it is not clear what to write in the ScalaDoc.
The comment you added doesn't say anything new. And the part "optinal additional (explorer-like) indexes" is better be added to ExtraIndex class, where it belongs (as it describes the index, not the serializer).
So, following you earlier suggestions, I would suggest to either to remove this ScalaDoc, or make it actually valuable.

Reply:
removed